mesheryctl/internal/cli/root/root.go

```diff
@@ -46,7 +46,7 @@ var (
 var RootCmd = &cobra.Command{
 	Use:   "mesheryctl",
 	Short: "Meshery Command Line tool",
-	Long:  `Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.`,
+	Long:  `Meshery is the Cloud Native management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.`,
```

Comment:
lowercase, please. Also, update the second "service mesh" reference in the sentence.

Reply:
lowercased and updated the reference.